lib/content-services/src/lib/search/services/search-configuration.service.ts

```diff
@@ -16,37 +16,32 @@
  */
 
 import { Injectable } from '@angular/core';
-import { QueryBody } from '@alfresco/js-api';
+import { SearchRequest } from '@alfresco/js-api';
 import { SearchConfigurationInterface } from '../../common/interfaces/search-configuration.interface';
 
 @Injectable({
     providedIn: 'root'
 })
 export class SearchConfigurationService implements SearchConfigurationInterface {
-
     /**
      * Generates a QueryBody object with custom search parameters.
      *
      * @param searchTerm Term text to search for
-     * @param maxResults Maximum number of search results to show in a page
+     * @param maxItems Maximum number of search results to show in a page
```

Comment:
Could you please update md file too for this service? 
Following changes need there: renaming maxResults to maxItems, renaming QueryBody to SearchRequest and correcting path to SearchRequest documentation instead of to QueryBody 
<img width="1056" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/5ec50968-2c0b-4be3-8734-82ad426bea70">